spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/ValidationMethod.scala

```diff
@@ -512,3 +512,315 @@ class MAE[@specialized(Float, Double)T: ClassTag]()
 
   override def format(): String = "MAE"
 }
+
+/**
+This validation method is used to obtain the performance in therms of recall:
+  tp / (tp + fn). This class requires a input class as parameter.
+  */
+class Recall[T: ClassTag](_class: T)
```

Comment:
Please not use variable name starting with `_`.
You can change `_class` to `classId`